src/pages/landing/jump-indices/_lazy-load.js

```diff
@@ -0,0 +1,6 @@
+import Loadable from '@loadable/component'
+
+export const DHero = Loadable(() => import('./_dHero.js'))
+export const DBanner = Loadable(() => import('./_dBanner.js'))
+export const JumpSection = Loadable(() => import('./_jumpSection.js'))
+export const Leverages = Loadable(() => import('./_leverages.js'))
```

Comment:
I don't think we'll be needing to do this as there are only 2 sections here and the users will probably need to see both of them straight on page load 😬

Reply:
Fixed in latest commit.